homeassistant/components/group/entity.py

```diff
@@ -425,6 +425,19 @@ def _async_update_group_state(self, tr_state: State | None = None) -> None:
         elif tr_state.attributes.get(ATTR_ASSUMED_STATE):
             self._assumed_state = True
 
+        registry: GroupIntegrationRegistry = self.hass.data[REG_KEY]
+
+        def _active_on_states() -> set[str]:
```

Comment:
Looking on my mobile, but it looks like it is only ever called in one place?

Reply:
It is, but I call it late as it is used under a condition.